daemon/algod/api/algod.oas2.json

```diff
@@ -1419,6 +1419,64 @@
         }
       }
     },
+    "/v2/teal/disassemble": {
```

Comment:
@winder - I expect this change to cause the `parity` job in indexer's nightly test to fail when the PR is merged. This can be fixed on Indexer's side by asserting the expected difference -as I expect that indexer wouldn't want to support this endpoint.

Reply:
Where do the asserts happen on the indexer tests?

Reply:
@algochoi I have a stop coming and didn't find all details.  Here's what I found.  Treat with a grain of salt because I haven't investigated the area before.

* https://github.com/algorand/indexer/search?q=parity - Points out the associated `Makefile` target.  It's ultimately run by `indexer-v-algod` in CircleCI.
* https://github.com/algorand/indexer/blob/b197dc35b836d8cf443dab7c6510bd1be6f089d0/misc/parity/test_indexer_v_algod.py - Appears to be the backing testing functionality.
* https://github.com/algorand/indexer/tree/b197dc35b836d8cf443dab7c6510bd1be6f089d0/misc/parity/reports - I _think_ a change is needed in `_full.yml`.  But, I ran out of time to fully evaluate.

Reply:
sorry.. I'll be glad to explain / fix on the indexer side when this breaks.